src/interactive/azext_interactive/azclishell/scenario_suggest.py

```diff
@@ -78,15 +118,25 @@ def try_remove(container, item):
                         if self.param_value_map[param]:
                             suggest.append(self.param_value_map[param])
                     return Suggestion(' '.join(suggest))
-            # If the user finish input with space, suggest all the rest unused parameters
             elif unfinished == '':
                 if not last_part.startswith('-'):
-                    suggest = []
+                    suggests = []
                     for param in unused_param:
-                        suggest.append(param)
-                        if self.param_value_map[param]:
-                            suggest.append(self.param_value_map[param])
-                    return Suggestion(' '.join(suggest))
+                        if param in self.special_global_param_map.keys():
+                            # sample_value is the sample values in scenarios, such as <RESOURCEGROUPNAME>
+                            sample_value = param
+                        else:
+                            sample_value = self.param_value_map[param]
+                        if sample_value:
+                            if sample_value in self.customized_param_value_map.keys():
+                                value = self.customized_param_value_map[sample_value]
```

Comment:
`sample_value` should only be one of the two cases, so it may not be accurate enough. May be the name `cached_param`  more appropriate?